randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -427,6 +427,9 @@ Extra - room_id: [13]
   * Extra - door_idx: (29,)
   > Room Center
       Trivial
+  > Event - Stabilizer 3 Online
+      # Speed Boost the Stabilizer: https://youtu.be/V20W0F4dZP0
+      Level 0 Locks and Speed Booster and After Stabilizer 5 Restored and Shinespark Tricks (Intermediate) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
you now have the wrong stabilizer event, should be 4. just double checked
![image](https://github.com/user-attachments/assets/852c8990-170f-4128-b7ae-ea29da630b84)

Reply:
I could have sworn i picked the right one. I must have accidentally scrolled on it